src/slackclient.cpp

```diff
@@ -123,15 +123,15 @@ void SlackClient::updatePresenceSubscription() {
             userIds.insert(channel.value("userId").toString());
         } else if (channel.value("type").toString() == "mpim" || channel.value("type").toString() == "group") {
             // TODO: Does presence of mpim and group ever makes sense
-            for (const auto &member : channel.value("memberIds").toList()) {
-                userIds.insert(member.toString());
-            }
+//            for (const auto &member : channel.value("memberIds").toList()) {
```

Comment:
Should I remove these lines? I don't think this is ever hit.

Reply:
Yep, I think you can.